src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,113 @@
+@import "./variables.scss";
```

Comment:
Is it necessary to add @import here? You've already written it in your main.scss file.